libs/facilities/frontend/view/src/lib/components/detail/detail.page.html

```diff
@@ -3,6 +3,8 @@
 		slot="header"
 		[headerTitle]="'Details of Facility ' + facility.heading"
 		headerSubtitle="On this page you can see all our data about this facility"
+		hasBackButton="true"
+		(backButtonClick)="goBack()"
```

Comment:
Why don't u include the logic directly inside the HTML file? The method doesn't do anything but pollute the TS file.

Reply:
I am just used to using methods but yeah it is unnecessary. With the directive that problem is solved anyways